### PR TITLE
Feature/make admin

### PR DESCRIPTION
**Pull request by mastrogiovanni**
# Description of change

I created a new command `make-admin`: this command will create a Job in the Kubernetes cluster where IS is deployed and this Job will update the role of the DID passed as parameter.
I created a new command `create-vc` to create verifiable credentials

## Links to any relevant issues

In order to be executed, this command requirements:
- client need to have a configured access to the Kubernetes cluster where IS is deployed.
- IS need to be installed via Helm chart (the command will retrieve MongoDB information from Helm chart descriptor) 

## Type of change

- Added a new command

## How the change has been tested

Build:
```
npm install
npm run build
```

Launch a minikube cluster with IS is a terminal:
```
minikube start
helm install is iota/integration-services --set apiKey=MYAPIKEY
kubectl port-forward svc/is-kong-proxy 3000:80
```

In another terminal

```
# Configure CLI
bin/is.js config -g http://localhost:3000 -k MYAPIKEY	

# Create issuer identity
bin/is.js create-identity<<EOF>issuer.json
{
    "username": "john.smith",
    "claimType": "Person",
    "claim": {
        "givenName": "John",
        "familyName": "Smith"
    }
}   
EOF

export ISSUER=$(jq -r .doc.id issuer.json)

# Make issuer "admin"
bin/is.js make-admin -i $ISSUER -d is


# Create a target identity
bin/is.js create-identity<<EOF>target.json
{
    "username": "john.smith.target",
    "claimType": "Person",
    "claim": {
        "givenName": "John",
        "familyName": "Smith"
    }
}   
EOF

export TARGET=$(jq -r .doc.id target.json)

# Create credential definition
cat <<EOF>credential.json
{
    "claimType": "Person",
    "credentialType": "BasicIdentityCredential",
    "claim": {
        "givenName": "Michele",
        "familyName": "Mastrogiovanni"
    }
}
EOF

# Create verifiable credential (it's possible only if Admin)
bin/is.js create-vc -i issuer.json -d $TARGET -c credential.json > vc.json

# Check VC
bin/is.js check-vc -i issuer.json vc.json
```

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: src/identity.ts [hunk 1]
```diff
@@ -148,6 +148,64 @@ export const removeTrustedAuthority = async (authorityId: string, options: { ide
...
+        if (!credentialType) {
+            console.log(chalk.bold.red("credentialType not present in credential file"));
+        }
+        if (!claimType) {
+            console.log(chalk.bold.red("claimType not present in credential file"));
+        }
+        if (!claim) {
+            console.log(chalk.bold.red("claim not present in credential file"));
+        }
+        if (error) {
```
Comment (dominic22) [inline src/identity.ts:182]: this is always false so it can be removed

File: src/identity.ts [hunk 2]
```diff
@@ -148,6 +148,64 @@ export const removeTrustedAuthority = async (authorityId: string, options: { ide
     }
 };
 
+export const parseInput = (fileName?: string) => {
+    let data = undefined;
+    if (!fileName) {
+        data = JSON.parse(fs.readFileSync(0, 'utf-8'));
+    }
+    else {
+        if (!fs.existsSync(fileName)) {
```
Comment (dominic22) [inline src/identity.ts:157]: would be cleaner to direcltly use and `else if(!fs.existsSync(fileName)) {` here

File: src/admin.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    if (!infos["DATABASE_URL"] || !infos["DATABASE_NAME"]) {
+        console.error(chalk.bold.green("Impossible to retrieve Integration Service info"))
+        console.error(chalk.bold.green("Please, be sure that Integration Services was installed"))
+        console.error(chalk.bold.green("using Helm Chart (https://github.com/iotaledger/helm-charts)"))
+        return;
+    }
+    const job = createMakeAdminJob(params.identity, infos["DATABASE_URL"], infos["DATABASE_NAME"]);
+    const k8sApi = k8s.KubernetesObjectApi.makeApiClient(kc);
+    await k8sApi.create(job);
+    console.error(chalk.bold.blue(`Identity ${params.identity} (if exists) is now Admin (i.e. it can issue verifiable credentials)`));
```
Comment (dominic22) [inline src/admin.ts:79]: why is this logged as error?

File: src/index.ts [hunk 4]
```diff
@@ -100,9 +107,19 @@ program
     .description('Remove trusted root identity by id')
     .action(removeTrustedAuthority);
 
+//create verifiable credential
+program
+    .command('create-vc')
+    .requiredOption('-i, --identityFile <path-to-identity-file>')
```
Comment (JuriBog) [inline src/index.ts:113]: you changed for the commands search-identity and find-identity the option description for -identityFile from <path-to-identity-file> to  <Identity file>. 
Here and and at the rest of the commands you let <path-to-identity-file>?

File: src/index.ts [hunk 5]
```diff
@@ -119,6 +136,7 @@ program
     .requiredOption('-i, --identityFile <path-to-identity-file>')
     .requiredOption('-t, --type <type-of-channel>')
     .requiredOption('-s, --source <source-of-channel>')
+    .option('-o, --outputFile <File where write the new Identity>')
```
Comment (JuriBog) [inline src/index.ts:139]: in the other commands you set it as required. maybe here too?

**Review by JuriBog — CHANGES_REQUESTED** [review]
I couldn't test the changes because I get one error when running 
`helm install is iota/integration-services --set apiKey=MYAPIKEY`
I get the error:
`Error: INSTALLATION FAILED: failed to download "iota/integration-services"`

File: src/admin.ts [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+/**
+ * Retrieve configuration of Integration Services.
+ * It need to be installed using helm-chart
+ */
+async function getISConfigMap(kc: KubeConfig, namespace: string, name: string) {
+    const k8sApiCore = kc.makeApiClient(k8s.CoreV1Api);
+    const response = await k8sApiCore.listNamespacedConfigMap(namespace)
+    return response.body.items.filter(item => {
+        const anns = item.metadata?.annotations;
+        if (anns && anns['meta.helm.sh/release-name'] && anns['meta.helm.sh/release-name'] === name) {
```
[the remark this answers is not in the document]
Comment (mastrogiovanni) [reply src/admin.ts:41]: right

File: src/identity.ts [hunk 7]
```diff
@@ -143,11 +131,53 @@ export const removeTrustedAuthority = async (authorityId: string, options: { ide
...
         console.log(chalk.bold.green(`Removed identity with identity id '${authorityId}' from trusted roots.`));
     } catch (e: any) {
-        console.log(chalk.bold.red(e.message));
-        if (e?.response?.data?.error) console.log(chalk.bold.red(e.response.data.error));
+        console.error(chalk.bold.red(e.message));
+        if (e?.response?.data?.error) console.error(chalk.bold.red(e.response.data.error));
     }
 };
 
+export const createCredential = async (options: { identityFile: string; did: string, credential?: string, outoutFile: string }) => {
```
[the remark this answers is not in the document]
Comment (mastrogiovanni) [reply src/identity.ts:139]: ok

File: src/identity.ts (hunk 1 above)
Comment (mastrogiovanni) [reply src/identity.ts:157]: missing "error = true"

File: src/index.ts (hunk 4 above)
Comment (mastrogiovanni) [reply src/index.ts:113]: I will add "path-to-identity-file" to all

File: src/index.ts (hunk 5 above)
Comment (mastrogiovanni) [reply src/index.ts:139]: right

**Review by JuriBog — APPROVED** [review]

**Review by dominic22 — APPROVED** [review]

**Pull request merged**